ui/src/app/views/cohort-edit/component.ts

```diff
@@ -3,52 +3,60 @@ import {Router, ActivatedRoute} from '@angular/router';
 
 import {Cohort} from 'generated';
 import {CohortsService} from 'generated';
-import {User} from 'app/models/user';
-import {UserService} from 'app/services/user.service';
-import {CohortEditService} from 'app/services/cohort-edit.service';
+import {WorkspaceComponent} from 'app/views/workspace/component';
 
 @Component({
   styleUrls: ['./component.css'],
   templateUrl: './component.html',
 })
 export class CohortEditComponent implements OnInit {
-  user: User;
   cohort: Cohort = {id: '', name: '', description: '', criteria: '', type: ''};
   cohortId: string;
   adding = false;
   constructor(
       private router: Router,
       private route: ActivatedRoute,
-      private userService: UserService,
       private cohortsService: CohortsService,
-      private CohortEditService: CohortEditService
   ) {}
 
   ngOnInit(): void {
-    this.userService.getLoggedInUser()
-        .then(user => this.user = user);
     this.cohortId = this.route.snapshot.url[4].path;
     if (this.route.snapshot.url[5] === undefined) {
```

Comment:
If we change the workspace link to /, this will break. I believe it changes to this.route.snapshot.url[3] here, and this.route.snapshot.url[2] above.